nixops/resources/ebs_volume.py

```diff
@@ -74,6 +75,14 @@ def show_type(self):
     def resource_id(self):
         return self.volume_id
 
+    def get_physical_spec(self):
+        physical = {}
+        if self.volume_id:
+            physical['volumeId'] = self.volume_id
```

Comment:
Any idea where this can be used ?

Reply:
I plan to use it to to tweak the show-physical-backup function to get  it to generate a file to override volumes with snapshots/other volumes